Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -133,53 +133,60 @@ union UDICFG
   explicit UDICFG(u32 hex) : Hex{hex} {}
 };
 
-// STATE_TO_SAVE
-
-// Hardware registers
-static UDISR s_DISR;
-static UDICVR s_DICVR;
-static u32 s_DICMDBUF[3];
-static u32 s_DIMAR;
-static u32 s_DILENGTH;
-static UDICR s_DICR;
-static u32 s_DIIMMBUF;
-static UDICFG s_DICFG;
-
-static StreamADPCM::ADPCMDecoder s_adpcm_decoder;
-
-// DTK
-static bool s_stream = false;
-static bool s_stop_at_track_end = false;
-static u64 s_audio_position;
-static u64 s_current_start;
-static u32 s_current_length;
-static u64 s_next_start;
-static u32 s_next_length;
-static u32 s_pending_samples;
-static bool s_enable_dtk = false;
-static u8 s_dtk_buffer_length = 0;  // TODO: figure out how this affects the regular buffer
-
-// Disc drive state
-static DriveState s_drive_state;
-static DriveError s_error_code;
-static u64 s_disc_end_offset;
-
-// Disc drive timing
-static u64 s_read_buffer_start_time;
-static u64 s_read_buffer_end_time;
-static u64 s_read_buffer_start_offset;
-static u64 s_read_buffer_end_offset;
-
-// Disc changing
-static std::string s_disc_path_to_insert;
-static std::vector<std::string> s_auto_disc_change_paths;
-static size_t s_auto_disc_change_index;
-
-// Events
-static CoreTiming::EventType* s_finish_executing_command;
-static CoreTiming::EventType* s_auto_change_disc;
-static CoreTiming::EventType* s_eject_disc;
-static CoreTiming::EventType* s_insert_disc;
+struct DVDInterfaceState::Data
+{
+  // Hardware registers
+  UDISR m_DISR;
+  UDICVR m_DICVR;
+  u32 m_DICMDBUF[3];
+  u32 m_DIMAR;
+  u32 m_DILENGTH;
+  UDICR m_DICR;
+  u32 m_DIIMMBUF;
+  UDICFG m_DICFG;
+
+  StreamADPCM::ADPCMDecoder m_adpcm_decoder;
+
+  // DTK
+  bool m_stream = false;
+  bool m_stop_at_track_end = false;
+  u64 m_audio_position;
+  u64 m_current_start;
+  u32 m_current_length;
+  u64 m_next_start;
+  u32 m_next_length;
+  u32 m_pending_samples;
+  bool m_enable_dtk = false;
+  u8 m_dtk_buffer_length = 0;  // TODO: figure out how this affects the regular buffer
+
+  // Disc drive state
+  DriveState m_drive_state;
+  DriveError m_error_code;
+  u64 m_disc_end_offset;
+
+  // Disc drive timing
+  u64 m_read_buffer_start_time;
+  u64 m_read_buffer_end_time;
+  u64 m_read_buffer_start_offset;
+  u64 m_read_buffer_end_offset;
+
+  // Disc changing
+  std::string m_disc_path_to_insert;
+  std::vector<std::string> m_auto_disc_change_paths;
+  size_t m_auto_disc_change_index;
+
+  // Events
+  CoreTiming::EventType* m_finish_executing_command;
+  CoreTiming::EventType* m_auto_change_disc;
+  CoreTiming::EventType* m_eject_disc;
+  CoreTiming::EventType* m_insert_disc;
```

Comment:
If I recall correctly, the Dolphin style is to not include the m_ prefix for public struct members.

Reply:
It doesn't actually explicitly say this one way or another, only that class member variables should have `m_` prefixes, and a struct and a class is kinda the same thing as far as C++ is concerned... but yeah I can change that, I don't really care either way.

Reply:
Yeah, I couldn't find it mentioned in Contributing.md. It's more of an unwritten convention.